sys/fs/cd9660/cd9660_vfsops.c

```diff
@@ -394,6 +403,23 @@ iso_mountfs(struct vnode *devvp, struct mount *mp)
 	vfs_flagopt(mp->mnt_optnew, "nojoliet", &isomp->im_flags, ISOFSMNT_NOJOLIET);
 	vfs_flagopt(mp->mnt_optnew, "kiconv", &isomp->im_flags, ISOFSMNT_KICONV);
 
+	if (1 == vfs_scanopt(mp->mnt_optnew, "uid", "%d", &v)) {
```

Comment:
Normal FreeBSD style here would be to use `... == 1` rather than `1 == ...`

Reply:
Fixed.